### ISSUE TITLE
Improved UI of Search Bar

### ISSUE THREAD

**Issue by Sarfraz-droid** [CircuitVerse/CircuitVerse#2662]
I would like to change the UI of Search Bar

Currently
![image 2](https://user-images.githubusercontent.com/73013838/144305220-8c4b9cf8-6aed-4f41-a844-f0f26fd4907f.png)

**Describe alternatives you've considered**
![Group 2](https://user-images.githubusercontent.com/73013838/144305235-ee7ef664-d3c3-4697-81a9-1a9d14aa8173.png)

**Are you working on this? Yes and I'll provide a PR soon**

**Comment by UkaniVedant** [comment]
Can you please assign me this issue.

**Comment by vedant-jain03** [comment]
@UkaniVedant 
@Sarfraz-droid is working on this issue. Kindly check different issue.

**Comment by UkaniVedant** [comment]
Ok no problem 👍🏻

**Comment by Snehagupta1907** [comment]
is anyone working on this issue? if not can I work on it

**Comment by ayan-biswas0412** [comment]
@Sarfraz-droid are you still working in this issue ?

**Comment by Sarfraz-droid** [comment]
Yes I am not working as for now.. I already did my changes and issued a pr for the same @Snehagupta1907 and @UkaniVedant can go ahead.

**Comment by kushalShukla-web** [comment]
is this issue still open

**Comment by vedant-jain03** [comment]
@Sarfraz-droid Any updates?
